server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaConsumer.ts

```diff
@@ -108,12 +110,14 @@ export class RdkafkaConsumer extends RdkafkaBase implements IConsumer {
 			"fetch.min.bytes": 1,
 			"fetch.max.bytes": 1024 * 1024,
 			"offset_commit_cb": true,
-			"rebalance_cb": this.consumerOptions.optimizedRebalance ? this.rebalance.bind(this) : true,
+			"rebalance_cb": this.consumerOptions.optimizedRebalance ?
+				(err: kafkaTypes.LibrdKafkaError, assignments: kafkaTypes.Assignment[]) => this.rebalance(consumer, err, assignments) :
+				true,
 			...this.consumerOptions.additionalOptions,
 			...this.sslOptions,
 		};
 
-		const consumer: kafkaTypes.KafkaConsumer = this.consumer =
+		consumer = this.consumer =
```

Comment:
I think the build is failing because it says `consumer.subscribe()` in line 127 below and `consumer.consume()` could possibly be undefined

`src/rdkafkaConsumer.ts:127:4 - error TS2532: Object is possibly 'undefined'.`